client/webserver/site/src/css/market.scss

```diff
@@ -19,7 +19,7 @@ div.marketlist {
 div.marketrow {
   font-size: 17px;
   padding: 0.3em;
-  font-family: 'demi-sans', sans-serif;
+  font-family: "demi-sans", sans-serif;
```

Comment:
I'm surprised stylelint didn't have an opinion about this.  We should decide on single/double and codify that with the `string-quotes`, `font-family-name-quotes`, `selector-attribute-quotes`, and `function-url-quotes` rules (e.g. https://github.com/stylelint/stylelint/blob/master/lib/rules/string-quotes/README.  **However**, it would be best to address this in a separate PR and leave all stylistic changes out of this.

e.g. **.stylelintrc**

```json
{
    "extends": "stylelint-config-standard",
    "rules": {
        "string-quotes": "double",
        "font-family-name-quotes": "always-unless-keyword",
        "selector-attribute-quotes": "always",
        "function-url-quotes": "always"
    }
}
```

I mention it because there are quite a few other inconsistencies if we intend to resolve them:

```
ERROR in 
src/css/application.scss
 14:8   ✖  Expected double quotes   string-quotes
 15:13  ✖  Expected double quotes   string-quotes
 16:8   ✖  Expected double quotes   string-quotes

src/css/icons.scss
 6:16  ✖  Expected double quotes   string-quotes

src/css/main.scss
  73:25  ✖  Expected double quotes            string-quotes            
 186:12  ✖  Expected quotes around "number"   selector-attribute-quotes

src/css/main_dark.scss
 29:27  ✖  Expected double quotes   string-quotes

src/css/typography.scss
  3:16  ✖  Expected double quotes   string-quotes
  5:9   ✖  Expected double quotes   string-quotes
  5:63  ✖  Expected double quotes   string-quotes
  6:9   ✖  Expected double quotes   string-quotes
  6:62  ✖  Expected double quotes   string-quotes
 13:16  ✖  Expected double quotes   string-quotes
 15:9   ✖  Expected double quotes   string-quotes
 15:55  ✖  Expected double quotes   string-quotes
 16:9   ✖  Expected double quotes   string-quotes
 16:54  ✖  Expected double quotes   string-quotes
 22:16  ✖  Expected double quotes   string-quotes
 24:9   ✖  Expected double quotes   string-quotes
 24:60  ✖  Expected double quotes   string-quotes
 25:9   ✖  Expected double quotes   string-quotes
 25:59  ✖  Expected double quotes   string-quotes
 32:16  ✖  Expected double quotes   string-quotes
 34:9   ✖  Expected double quotes   string-quotes
 34:44  ✖  Expected double quotes   string-quotes
 35:9   ✖  Expected double quotes   string-quotes
 35:45  ✖  Expected double quotes   string-quotes
 36:9   ✖  Expected double quotes   string-quotes
 36:52  ✖  Expected double quotes   string-quotes
```

Reply:
The suggested rule additions listed here might be worth reviewing too: https://github.com/stylelint/stylelint-config-standard#suggested-additions

Reply:
Agreed, will revert these changes here and can tweak our linters in another PR.